test/lib/ex_debug_toolbar/collector/instrumentation_collector_test.exs

```diff
@@ -19,11 +22,50 @@ defmodule ExDebugToolbar.Collector.InstrumentationCollectorTest do
     end
 
     test "it records request timeline on stop" do
-      Collector.ex_debug_toolbar(:start, %{}, %{})
-      Collector.ex_debug_toolbar(:stop, %{}, %{})
+      call_collector(&Collector.ex_debug_toolbar/3, %{})
       assert {:ok, request} = get_request(@request_id)
       assert request.data.timeline.events |> Enum.any?
       assert request.data.timeline.duration > 0
     end
   end
+
+  describe "phoenix_controller_call"  do
+    test "it starts a phoenix_controller event" do
+      Toolbar.start_request
```

Comment:
maybe something like this to clean up a bit 
```
setup :start_request

defp start_request do
 Toolbar.start_request
end
```

`start_request/0` could even be put to request helpers